Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/EntityConfigurations/RelationshipAuditLogEntryEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Relationships;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Database.EntityConfigurations;
+public class RelationshipAuditLogEntryEntityTypeConfiguration : IEntityTypeConfiguration<RelationshipAuditLogEntry>
+{
+    public void Configure(EntityTypeBuilder<RelationshipAuditLogEntry> builder)
+    {
+        builder.ToTable(nameof(RelationshipAuditLogEntry), "RelationshipAuditLog", x => x.ExcludeFromMigrations());
```

Comment:
Hmm. I guess you never ever manually tested your changes. Correct? Because I'm pretty sure this will not work. The table is not called "RelationshipAuditLogEntry".

Reply:
I have to be honest I didn't test the endpoint beforehand. I believed Nikola 2 that this is working after he explained all parts of the code. We sat down today an fixed it and tested manually. Now it is working. Changes made.